### PR TITLE
fix: missing semicolon

### PR DESCRIPTION
**Pull request by robothot**
### 🤔 fix missing semicolon 

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |    修复缺少分号       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-import { cloneElement } from '../_util/reactNode';
-import getPlacements from '../_util/placements';
-import OverrideContext from '../menu/OverrideContext';
 import type { OverrideContextProps } from '../menu/OverrideContext';
+import OverrideContext from '../menu/OverrideContext';
+import getPlacements from '../_util/placements';
+import { cloneElement } from '../_util/reactNode';
+import { tuple } from '../_util/type';
+import warning from '../_util/warning';
+import DropdownButton from './dropdown-button';
```
Comment (yoyo837) [inline components/dropdown/dropdown.tsx:12]: 为啥动上面的顺序，合并处理冲突的话好累的。😂

Comment (robothot) [reply components/dropdown/dropdown.tsx:12]: 我拉代码，然后执行了一下 npm install 然后执行 git push上面的顺序就会自动变化, 我是window 电脑， 我也好奇怪、

Comment (robothot) [reply components/dropdown/dropdown.tsx:12]: 我自己测试了一下， 和

✍️  Fixing up components/anchor/Anchor.tsx.

这个有关系

Comment (yoyo837) [reply components/dropdown/dropdown.tsx:12]: ide干的？

Comment (robothot) [reply components/dropdown/dropdown.tsx:12]: 和 husky 有关， 下面我没有确定， 似乎每次 commit 的时候都会按照顺序进行把 import 进行调整

Comment (yoyo837) [reply components/dropdown/dropdown.tsx:12]: prettier？

Comment (robothot) [reply components/dropdown/dropdown.tsx:12]: 是的 npx --no-install pretty-quick --staged, 执行了这个命令把顺序进行调整了

**Pull request merged**